tanner/emulators/cmd_exec.py

```diff
@@ -0,0 +1,73 @@
+import asyncio
+import docker
+import logging
+
+from tanner.config import TannerConfig
+from tanner.utils import patterns
+
+class CmdExecEmulator:
+	def __init__(self):
+		self.docker_client = docker.from_env(version='auto')
```

Comment:
So that will be done in documentation right? I'll do in a separate PR or should I do it here itself?